src/lib/enums.js

```diff
@@ -13,7 +13,11 @@ export const claimStages = {
       "REJECTED": 3
   }
 
-  export const SafeType = {
-    "CRYPTO": 0,
-    "GENERIC": 1
-  }
\ No newline at end of file
+  export const NetworkType = {
```

Comment:
Let's refactor all the enums in this file. We do not need to define the value for enums. This one should be:
```
export enum NetworkType {
mainnet,
 testnet,
 local,
devnet
}
```
Use all small for the enum values. And network type should have only these values.

Refactor all the enums in this file and maintain the naming consistency.